### DISCUSSION TITLE
Может кто настроенный под youtube/discord на NFQWS blockcheck скинуть?

### DISCUSSION THREAD

**Discussion by ThisIsNoTa2acc** [bol-van/zapret#807] [Q&A]
Я вроде пару недель назад пробывал сам ковыряться, но выдавало совершенно нерабочие стратегии, а щас роскомпозор опять начинает мои дикие костыли блокировать потихоньку.

**Comment by NewUse** [comment]
во-первых, есть готовый кастом для дискорд, идёт из коробки , только скопировать в соотв. папку
во-вторых нет смысла выкладывать стратегии -- они индивидуальны для конкретного провайдера 
в-третьих в дискуссиях есть соотв. обсуждения с примерами
в-четвёртых -- никаких гарантий, что оно будет работать
